spx-gui/src/apis/util.ts

```diff
@@ -22,6 +22,22 @@ export function getUpInfo() {
 }
 
 export async function makeObjectUrls(objects: UniversalUrl[]): Promise<UniversalToWebUrlMap> {
-  const result = (await client.post('/util/fileurls', { objects: objects })) as { objectUrls: UniversalToWebUrlMap }
-  return result.objectUrls
+  return workAroundIssue1598(objects)
+
+  // const result = (await client.post('/util/fileurls', { objects: objects })) as { objectUrls: UniversalToWebUrlMap }
+  // return result.objectUrls
+}
+
+/** Workaround for https://github.com/goplus/builder/issues/1598 */
+const universalUrlPrefixToWebUrlPrefix: Record<UniversalUrl, WebUrl> = {
```

Comment:
For a temporary fix, it may be simpler to use `usercontentBaseUrl` from the environment directly, bypassing bucket name checks since we expect only one bucket per environment. This avoids hardcoding bucket & domain details.

Reply:
done https://github.com/goplus/builder/compare/5f1d109d5f827b17b7d9399406e7aab07b45deb7..195ba9204d0aaab50fb6608b0f0009b0e0c27296